modules/300-prometheus/images/grafana-dashboard-provisioner/hooks/dashboard_provisioner.sh

```diff
@@ -47,6 +48,24 @@ function __main__() {
 
     title=$(slugify <<<${title})
 
+    dashboardUid=$(jq -rc '.definition | try(fromjson | .uid)')
+    check_dashboard=0
+
+    if [[ ! -s "${tmpFile}" ]]; then
+      echo "${dashboardUid}" >> "${tmpFile}"
+    else
+      for uid in $(cat "${tmpFile}"); do
+        if [[ "${uid}" == "${dashboardUid}" ]]; then
+          check_dashboard=1
+        fi
+      done
+      if [[ "${check_dashboard}" -eq 0 ]]; then
+        echo "${dashboardUid}" >> "${tmpFile}"
+      else
+        echo "The dashboard with specified uid exists: ${dashboardUId}"
```

Comment:
I think we need the `continue` word or somethink like that here. To avoid further logic on this iteration

Reply:
@yalosev, it was done. Please, check now.